kod/object/passive/spell/defile.kod

```diff
@@ -49,14 +49,30 @@ classvars:
 
    viSpell_level = 3
    viMana = 0
-   viMana_bonus = 10
 
    viCast_time = 10000
    viChance_to_Increase = 100
```

Comment:
This should be much lower...this is a guaranteed improve (I could've sworn chance to increase was 50?).

Reply:
Good catch. Updated to 50.